AtlaTeam/ojs
# Patrick add child theme

Created a new branch **atla-develop** off the **atla_3_3_0_8** branch. Plan to use this branch to contain Atla changes moving forward. For next PKP release, let's rebase off this branch.

To Test:

* If checking out branch in existing serials-dev directory, remember to delete the ../plugins/themes/bootstrap3 directory.
* Check out the working branch. 
`git checkout patrick-add-child-theme`
`git pull`
* Install submodules and verify that bootstrap3 has been successfully added to ../plugins/themes.
`git submodule update --init --recursive`
* Navigate to <BASE_URL>/admin/settings#plugins and check the "Atla Bootstrap3 Theme" checkbox at the bottom of the list.
* Navigate to <BASE_URL>/admin/settings#appearance and select "Atla Bootstrap3 Theme" from the dropdown. Select the "Default Bootstrap Theme" from the list.
* Go the home page and confirm that the page displays correctly with Atla's color scheme (orange buttons). You may need to clear caches.
* Navigate to <BASE_URL>/admin/wizard/6#plugins and check the "Atla Bootstrap3 Theme" checkbox at the bottom of the list.
* Navigate to <BASE_URL>/admin/wizard/6#setup/appearance and select "Atla Bootstrap3 Theme" from the dropdown. Select the "Default Bootstrap Theme" from the list (unless another option is already selected).
* Repeat the above two steps for the remaining journals. I believe one or two journals, such as the Wabash one, may use a different Bootstrap subtheme ("Journal" in the case of Wabash).
